src/core/settings.py

```diff
@@ -240,61 +251,70 @@ def ugettext(s):
 
 # 1.9 appears confused about where null and blank are required for many to
 # many fields, so we're hiding these warning from the console
+# TODO: code smell, revisit
 SILENCED_SYSTEM_CHECKS = (
     'fields.W340',
 )
 
-'''
-# This section should only be enabled if you intend to use Sentry for error reporting.
 LOGGING = {
     'version': 1,
     'disable_existing_loggers': True,
+    # TODO: do we really need to be affecting the root logger here?
```

Comment:
The Sentry documentation suggest, yes. https://docs.sentry.io/clients/python/integrations/django/

Reply:
yeah, dead comment. will remove